automation/test.sh

```diff
@@ -324,6 +324,9 @@ spec:
 EOF
 fi
 
+if [[ "$KUBEVIRT_STORAGE" == "rook-ceph" ]]; then
```

Comment:
This is going to become a little bit confusing with KUBEVIRT_STORAGE overriding focuses from above. We should address that later. Good enough for now.